Core/Core/Database/CoreData/Models/Module.swift

```diff
@@ -23,6 +23,7 @@ public class Module: NSManagedObject {
     @NSManaged public var position: Int
     @NSManaged public var courseID: String
     @NSManaged public var published: Bool
+    @NSManaged public var items: Set<ModuleItem>?
```

Comment:
aren't the items supposed to be ordered?

Reply:
I was thinking I would order them via the NSFetchedResultsController. I don't think they need to be ordered here.

Would I just use a `NSOrderedSet`?

Reply:
if we aren't going to access them from this, why create the relationship?

Reply:
Good point. I added it for a test. I will remove the relationship.

Reply:
Replaced relationship with `courseID` reference.